packages/eslint-plugin-slds/src/rules/enforce-bem-class.ts

```diff
@@ -28,7 +28,7 @@ export = {
 
   create(context) {
     const pattern =
-      /^(?:[a-z0-9]+(?:-[a-z0-9]+)*)(__[a-z0-9]+(?:-[a-z0-9]+)*)?(--[a-z0-9]+(?:-[a-z0-9]+)*)?$/;
+   /^([a-z]+|[a-z]+-[a-z]+|slds-[a-z0-9]+(-[a-z0-9]+)*(--[a-z0-9]+(-[a-z0-9]+)*)?(__[a-z0-9]+)?)$/;
```

Comment:
@kenamverma Would you be able to give some examples for positive and negative cases on this regex. That would help someone who want to debug later.